### PR TITLE
Fix SpeechInputExample scene (missing geometry)

### PR DESCRIPTION
**Pull request by cre8ivepark**
## Overview
Fixing missing geometry in the Speech Input Example scene.
Updated typo.

2021-05-03 00_46_00-MRTK - SpeechInputExamples - Universal Windows Platform - Unity 2019 4 22f1  PRE

2021-05-03 00_46_16-MRTK - SpeechInputExamples - Universal Windows Platform - Unity 2019 4 22f1  PRE

### CODE REVIEW

File: ProjectSettings/EditorSettings.asset [hunk 1]
```diff
@@ -3,19 +3,33 @@
...
 EditorSettings:
   m_ObjectHideFlags: 0
-  serializedVersion: 7
+  serializedVersion: 9
   m_ExternalVersionControlSupport: Visible Meta Files
   m_SerializationMode: 2
   m_LineEndingsForNewScripts: 2
   m_DefaultBehaviorMode: 0
+  m_PrefabRegularEnvironment: {fileID: 0}
+  m_PrefabUIEnvironment: {fileID: 0}
```
Comment (CDiaz-MS) [inline ProjectSettings/EditorSettings.asset:12]: Are the changes to the EditorSettings, ProjectSettings, and QualitySettings asset files intentional?

Comment (cre8ivepark) [reply ProjectSettings/EditorSettings.asset]: reverted the changes

**Review by RogPodge — APPROVED** [review]

**Pull request merged**